src/main/java/com/launchdarkly/client/LDClient.java

```diff
@@ -179,16 +181,16 @@ public boolean getFlag(String featureKey, LDUser user, boolean defaultValue) {
   }
 
   /**
-   * Returns a map from feature flag keys to boolean feature flag values for a given user. The map will contain {@code null}
+   * Returns a map from feature flag keys to {@link JsonElement} feature flag values for a given user. The map will contain {@code null}
    * entries for any flags that are off. If the client is offline or has not been initialized, a {@code null} map will be returned.
    * This method will not send analytics events back to LaunchDarkly.
-   *
+   * <p>
    * The most common use case for this method is to bootstrap a set of client-side feature flags from a back-end service.
    *
    * @param user the end user requesting the feature flags
-   * @return a map from feature flag keys to boolean feature flag values for the specified user
+   * @return a map from feature flag keys to JsonElement values for the specified user
    */
-  public Map<String, Boolean> allFlags(LDUser user) {
+  public Map<String, JsonElement> allFlags(LDUser user) {
```

Comment:
Hmm. I think we should retain a helper that fetches all boolean flags. That's likely the most common use case for this method, and it's inconvenient to have to transform the map to filter out just the boolean flags. 

Whatever we do we should square this away so that all the typed languages have the same API footprint. The untyped languages should be similarly consistent. Should probably sync up on a separate Quip doc on that.